src/app/not-found/not-found.component.spec.ts

```diff
@@ -31,7 +31,9 @@ import { DropdownComponent } from '../dropdown/dropdown.component';
 import {IntelligenceComponent} from "../intelligence/intelligence.component";
 import {SpeechtotextComponent} from "../speechtotext/speechtotext.component";
 import {AutoCorrectComponent} from "../auto-correct/auto-correct.component";
+import {StatsboxComponent} from "../statsbox/statsbox.component";
 import {InfiniteScrollModule} from "ngx-infinite-scroll";
+import {ChartsModule} from "ng2-charts";
 
```

Comment:
we are importing all modules into NotFound component spec it has all components in it.